hyper/http11/response.py

```diff
@@ -48,7 +49,8 @@ def __init__(self, code, reason, headers, sock, connection=None):
         # bother checking for content-length, we just keep reading until
         # we no longer can.
         self._expect_close = False
-        if b'close' in self.headers.get(b'connection', []):
+        if (b'close' in self.headers.get(b'connection', []) or
+                b'close' in self.headers.get(b'proxy-connection', [])):
             self._expect_close = True
```

Comment:
But, if we ignore this header, how should we deal with that assertion then?

I believe that such response might violate a standard, but this one I get from a real proxy server, and I'm pretty sure that this is not a single example out there.

So I guess we should work-around such responses somehow. I thought that treating `proxy-connection: close` like a `connection: close` was a decent solution.